installer.nsh

```diff
@@ -20,69 +24,23 @@
     
     FileWrite $installLog "$\nInstall (${__DATE__} ${__TIME__}): $\r$\n"
     FileWrite $installLog "-----------------------------$\r$\n"
-     
-    Var /GLOBAL VERSION
-    FileOpen $4 "$INSTDIR\resources\cli-version.txt" r
-    FileRead $4 $VERSION
-    FileClose $4
 
-    Var /GLOBAL PLATFORM
     ${If} ${RunningX64}
-        StrCpy $PLATFORM "remoteit_windows_x86_64.exe"
         FileWrite $installLog "- Platform X64$\r$\n"
+        StrCpy $REMOTE_CLI_EXE "$BINARIES\x64\remoteit.exe"
+        StrCpy $MUXER "$BINARIES\x64\muxer.exe"
+        StrCpy $DEMUXER "$BINARIES\x64\demuxer.exe"
+        StrCpy $CONNECTD "$BINARIES\x64\connectd.exe"
```

Comment:
We don't want to copy anything here, just add to the windows PATH so that the `remoteit` command will work without the full path.